plugins/modules/clustering/consul/consul_session.py

```diff
@@ -95,6 +95,11 @@
         choices: [ delete, release ]
         type: str
         default: release
+    ttl:
+        description:
+          - Specifies the duration of a session (between 10 and 86400).
```

Comment:
Would be good to also mention the unit :)
```suggestion
          - Specifies the duration of a session in seconds (between 10 and 86400).
```